source/Locales/PatchProperties.lagda

```diff
@@ -642,6 +655,85 @@ module Complementation (X : Locale (𝓤 ⁺) 𝓤 𝓤) (σᴰ : spectralᴰ (
 
 \end{code}
 
+I define below an alternative version of the module above due to a technical
+problem. Consider a spectral locale `A` with a proof `σ` of spectrality. When
+the module above is called with the data contained in `σ` (let's call this
+`σᴰ`), Agda does not recognise the fact that `σ = ∣ σᴰ ∣`. To circumvent this
+problem, I'm defining a version of this module that takes as argument the proof
+of spectrality instead of the structure contained within.
+
+\begin{code}
+
+module PatchComplementationAlternative (X : Locale (𝓤 ⁺) 𝓤 𝓤)
```

Comment:
This module already exists elsewhere, Ayberk tells me, and so should be deleted.

Reply:
I'll do this later.